### ISSUE TITLE
tradfri-status error

### ISSUE THREAD

**Issue by mkierczak** [hvanderlaan/ikea-smartlight#26]
While tradfri-lights works perfectly well, the tradfri-status returns:
```
./tradfri-status.py
[ ] Tradfri: acquiring all Tradfri devices, please wait ...
Traceback (most recent call last):
  File "./tradfri-status.py", line 95, in <module>
    main()
  File "./tradfri-status.py", line 52, in main
    devices = tradfriStatus.tradfri_get_devices(hubip, apiuser, apikey)
  File "/home/pi/src/ikea-smartlight/tradfri/tradfriStatus.py", line 46, in tradfri_get_devices
    return json.loads(result.read().strip('\n'))
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```
A bit cryptic error...

**Comment by hvanderlaan** [comment]
This could be the case when your device has a strange name / or special characters.
When I run the command it all works fine.

```
% ./tradfri-status.py                                  ±[devel] ~/Projects/ikea-smartlight
[ ] Tradfri: acquiring all Tradfri devices, please wait ...
Tradfri devices: 100%|█████████████████| 10/10 [00:00<00:00, 46.86 devices/s]
Tradfri groups: 100%|█████████████████| 5/5 [00:00<00:00, 27.39 group/s]
[+] Tradfri: device information gathered
===========================================================

bulb ID 65540, name: lamp hal boven, brightness: 7, state: off
bulb ID 65542, name: lamp slaapkamer, brightness: 22, state: on
bulb ID 65541, name: lamp hal beneden, brightness: 5, state: off
bulb ID 65538, name: lamp bank, brightness: 254, state: on
bulb ID 65545, name: lamp schuur 1, brightness: 254, state: off
bulb ID 65546, name: lamp schuur 2, brightness: 254, state: off
bulb ID 65539, name: lamp kastje, brightness: 254, state: on
bulb ID 65537, name: lamp tv, brightness: 254, state: on


group ID: 137274, name: hal beneden, state: off
group ID: 186970, name: hal boven, state: off
group ID: 186972, name: buitenlanpen, state: off
group ID: 183216, name: slaapkamer, state: off
group ID: 140387, name: woonkamer, state: off```

**Comment by hvanderlaan** [comment]
pull request #27 will break my setup. This could be an issue with the tradfri version.
Could you please post your gateway version? than i can compare it with my version.

```[ ] Tradfri: acquiring all Tradfri devices, please wait ...
Traceback (most recent call last):
  File "./tradfri-status.py", line 95, in <module>
    main()
  File "./tradfri-status.py", line 52, in main
    devices = tradfriStatus.tradfri_get_devices(hubip, apiuser, apikey)
  File "/Users/haraldvdlaan/Projects/ikea-smartlight/tradfri/tradfriStatus.py", line 47, in tradfri_get_devices
    return json.loads(result.read().split('\n')[3])
IndexError: list index out of range```

**Comment by hvanderlaan** [comment]
This should be fixed with #29 Please try again
